### PR TITLE
Made the changes Dan suggested

### PR DESCRIPTION
**Pull request by sepiatonal**

### CODE REVIEW

File: src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java [hunk 1]
```diff
@@ -21,13 +20,12 @@ public MobSpawnListener() {
 	@EventHandler(priority = EventPriority.LOW, ignoreCancelled = false)
 	public void onMobSpawn(SpawnerSpawnEvent e) {
 		e.setCancelled(true);
-		e.getSpawner().setDelay(ConfigManager.getHarvestInterval());
+		e.getSpawner().setDelay(Integer.MAX_VALUE);
```
Comment (ProgrammerDan) [inline src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java:23]: Not sure I understnd the change to MAX_VALUE here -- what triggers the XP drop now?

File: src/main/java/com/github/soerxpso/xpspawners/manager/SpawnerManager.java [hunk 2]
```diff
@@ -10,57 +9,41 @@
...
-				for(Spawner s : chunk) {
-					if(closest == null) {
-						closest = s;
-						continue;
-					}
-					if(loc.distance(s.getLocation()) < loc.distance(closest.getLocation())) {
-						closest = s;
-					}
-				}
+		Set<QTBox> nearby = spawners.find(loc.getBlockX(), loc.getBlockZ());
```
Comment (ProgrammerDan) [inline src/main/java/com/github/soerxpso/xpspawners/manager/SpawnerManager.java:25]: Did you confirm that the find returns nearest?

File: src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java (hunk 1 above)
Comment (sepiatonal) [reply src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java:23]: Oops. The Spawner class got unadded somehow. I fixed it.

There's now a Bukkit TimerTask in the Spawner class that schedules drops. Using the SpawnerSpawnEvent without any sort of timer was causing occasional double drops.

File: src/main/java/com/github/soerxpso/xpspawners/manager/SpawnerManager.java (hunk 2 above)
Comment (sepiatonal) [reply src/main/java/com/github/soerxpso/xpspawners/manager/SpawnerManager.java:25]: Find returns a set of all QTBoxes within range (a 33x33x33 cube). I'm looping through the set afterward to check which is nearest.

**Pull request merged**